bodhi/server/consumers/masher.py

```diff
@@ -938,7 +938,7 @@ def compose_atomic_trees(self):
                 notifications.publish(topic="ostree.compose.fail",
                                       msg=dict(tag=tag),
                                       force=True)
-                raise Exception('%s atomic compose failed' % tag)
+                self.log.error('%s atomic tree compose failed', tag)
```

Comment:
As you might expect, we don't have test coverage for this line. I volunteer to write tests, but it may take me some time.